CommandDotNet.Example/Program.cs

```diff
@@ -17,7 +17,10 @@ static int Main(string[] args)
             AppRunner<TApp> appRunner = new AppRunner<TApp>(new AppSettings()
             {
                 Case = Case.KebabCase,
-                HelpTextStyle = HelpTextStyle.Detailed
+                Help =
+                {
+                    TextStyle = HelpTextStyle.Detailed
+                }
```

Comment:
I really like the updates they've made to the object ctors.  I think this a good way to group related settings.  Viewing them in logical groupings makes it easier to grok.